proxy/predeployed/test/generate_genesis.py

```diff
@@ -5,32 +5,36 @@
 
 
 def main():
-    if len(sys.argv) < 3:
+    if len(sys.argv) < 4:
```

Comment:
It is an external change, we need to notify Skale-node about this change.

Reply:
is this necessary? @payvint @yavrsky @DimaStebaev